pwiz_tools/Skyline/Model/Tools/PythonInstaller.cs

```diff
@@ -1076,13 +1077,13 @@ public RunGetPipScriptTask(PythonInstaller installer)
         public override void DoAction(ILongWaitBroker broker)
         {
             // CONSIDER(brendanx): Path quoting?
-            var cmdLine = TextUtil.SpaceSeparate(PythonInstaller.BasePythonExecutablePath,
-                PythonInstaller.GetPipScriptDownloadPath);
+            var cmdLine = TextUtil.SpaceSeparate(TextUtil.Quote(PythonInstaller.BasePythonExecutablePath),
```

Comment:
Note that the function prototype:

public static string Quote(this string text)

The "this" means you can just use:

PythonInstaller.BasePythonExecutablePath.Quote()